docs/notebooks/data_transformation.pct.py

```diff
@@ -0,0 +1,227 @@
+        tf.math.log(kernel.lengthscales), prior_scale
+    )
+    gpr = gpflow.models.GPR(
+        data.astuple(), kernel, mean_function=gpflow.mean_functions.Constant(), noise_variance=1e-5
+    )
+    gpflow.set_trainable(gpr.likelihood, False)
+
+    return GaussianProcessRegression(
+        model=gpr,
+        optimizer=Optimizer(
+            gpflow.optimizers.Scipy(), minimize_args={"options": dict(maxiter=100)}
+        ),
+        num_kernel_samples=100,
+    )
+
+
+# build the model
+model = build_gp_model(initial_data)
+
+
+# %% [markdown]
+# ## Run the optimization loop
+#
+# We can now run the Bayesian optimization loop by defining a `BayesianOptimizer` and calling its `optimize` method.
+#
+# The optimizer uses an acquisition rule to choose where in the search space to try on each optimization step. We'll be using Expected improvement acquisition function - it is used by default, so no need to specify it.
+#
+# We'll run the optimizer for 100 steps. Note: this may take a while!
+
+# %%
+num_acquisitions = 100
+
+bo = trieste.bayesian_optimizer.BayesianOptimizer(observer, search_space)
+result = bo.optimize(num_acquisitions, initial_data, model)
+dataset = result.try_get_final_dataset()
+
+
+# %% [markdown]
+# ## Explore the results
+#
+# We can now get the best point found by the optimizer. Note this isn't necessarily the point that was last evaluated. We will also plot regret for each optimization step.
+#
+# We can see that the optimization did not get close to the global optimum of -210.
+
+# %%
+query_points = dataset.query_points.numpy()
+observations = dataset.observations.numpy()
+
+arg_min_idx = tf.squeeze(tf.argmin(observations, axis=0))
+
+print(f"query point: {query_points[arg_min_idx, :]}")
+print(f"observation: {observations[arg_min_idx, :]}")
+
+
+# %% [markdown]
+# Once ask-tell optimization is over, you can extract an optimization result object and perform whatever analysis you need, just like with regular Trieste optimization interface. For instance, here we will plot regret for each optimization step.
+
+# %%
+def plot_ask_tell_regret(dataset):
+    observations = dataset.observations.numpy()
+    arg_min_idx = tf.squeeze(tf.argmin(observations, axis=0))
+
+    suboptimality = observations - F_MINIMUM.numpy()
+    ax = plt.gca()
+    plot_regret(suboptimality, ax, num_init=num_initial_points, idx_best=arg_min_idx)
+
+    ax.set_yscale("log")
+    ax.set_ylabel("Regret")
+    ax.set_ylim(0.001, 100000)
+    ax.set_xlabel("# evaluations")
+
+
+plot_ask_tell_regret(dataset)
+
+
+# %% [markdown]
+# # Data transformation with the help of Ask-Tell interface
+#
+# We first write a simple function for doing the standardisation of the data, that is, we scale the data to have a zero mean and a variance equal to 1. We also return the mean and standard deviation parameters as we will use them to transform new points.
+
+# %%
+def standardise(x, mean=None, std=None):
```

Comment:
So, shall we call whatever is going on with the data in this notebook "normalization" (as in PR title) or "standardisation" (as in here)? Probably good to just stick to one name throughout